policies/comms.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Mainflux
```

Comment:
We discussed this should not be needed - policies we'll never communicate with agents directly. It looks like it may be unused and is old code?